lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -29,7 +29,19 @@
 
 <li class="course-item">
   % if settings.FEATURES.get('ENABLE_VERIFIED_CERTIFICATES'):
-    <article class="course ${enrollment.mode}">
+    % if enrollment.mode == "verified":
+      % if settings.FEATURES.get('SEPARATE_VERIFICATION_FROM_PAYMENT') and request.session.get('separate-verified', False):
+        % if verification_status.get('status') in [VERIFY_STATUS_NEED_TO_VERIFY, VERIFY_STATUS_SUBMITTED, VERIFY_STATUS_APPROVED]:
+          <article class="course verified">
+        % else:
+          <article class="course honor">
+        % endif
+      % else:
+        <article class="course verified">
+      % endif
+    % else:
+      <article class="course ${enrollment.mode}">
+    % endif
```

Comment:
From offline discussions with @wedaly and @griffresch: we don't currently use the verification status to manipulate the banner message for professional education courses; this is not a concern for now, and as such we won't manipulate the banner color for these courses either.